Customer routes done

wvandolah opened the pull request:
- [x] move create user db functions out of google.js and jwt.js
- [x] test token gen
- [x] finish redirect route
- [x] rewrite customer routes so they look more like prod routes. 
- [x] need routes for after a cust has authed with google to input remain field in db (address, email, phone)

wvandolah committed:
started on customer auth, super messy

wvandolah committed:
google auth less awful looking

wvandolah committed:
cleaned up jwt

wvandolah committed:
fixed issue with token never expiring

NandoTheessen commented on auth/google.js:
```diff
@@ -10,34 +14,21 @@ const passportConfig = {
+    console.log(profile)
+    try {
+        const { id, displayName } = profile;
+        // need to consider changing db to hold all emails
```
I really like the refactor of this William, good job! 
If I could improve something, it would be more comments in your code.
As we are working with quite a few people those will probably come in handy to teach your colleagues about passport / make explaining easier down the road.

NandoTheessen commented on auth/google.js:
```diff
@@ -10,34 +14,21 @@ const passportConfig = {
+        const { id, displayName } = profile;
+        // need to consider changing db to hold all emails
+        const email = profile.emails[0].value;
+        const searchUser = await findUser(id);
```
Unhandled promise rejections are deprecated in node (fancy speak for: either use a try / catch block or .catch if you are "await"ing something )

Mister-Corn committed:
Change customers table and create customer_email table

Change customers table to have the ID received from Google OAuth to be the primary key of the customers table.

Add customer_email table to allow a user to be associated with multiple e-mail addresses.

wvandolah committed:
Merge pull request #1 from Mister-Corn/master

Change customers table and create customer_email table

NandoTheessen commented:
What is needed to completed? Do you need a helping hand with this?

wvandolah commented:
I need to change a few things.  Me and ronald decided to make some changes to the db. so i need to change some of the customer routes

wvandolah committed:
added cust routes, commented auth routes
